Harvest.Web/ClientApp/src/App.tsx

```diff
@@ -100,7 +100,7 @@ function App() {
         path="/expense/unbilled/:projectId"
         component={UnbilledExpensesContainer}
       />
-      <Route path="/home/map" component={Map} />
+      <Route exact path="/project/map" component={ProjectFields}/>
```

Comment:
I think this should be using the conditional route. Something like this
`      <ConditionalRoute
        exact
        roles={["FieldManager"]}`

Reply:
Based on Spruce's other reply. Worker, FM, and supervisor... Not sure what that looks like.

Reply:
It's possible to add multiple authorize policies to one route correct?

Reply:
@jSylvestre Do you think it should be `"FieldManager"`? I was thinking `"Worker"`

Reply:
I think just Supervisor/FieldManager

Reply:
Later we might re-use the basics of this component for a worker facing page but doesn't need to be this one